kitten-star/r1.17.0/build.gradle.kts

```diff
@@ -0,0 +1,9 @@
+plugins {
+    id("kit_tunes.java.17")
```

Comment:
I just realized I didn't use the Java 16 plugin for this, can you fix that @Pixaurora?

Reply:
I fixed it locally, next time I push changes it'll be fixed! Thank you for pointing that out